src/gen/templates/fixNames.js

```diff
@@ -0,0 +1,36 @@
+const { Project, StructureKind } = require("ts-morph");
+
+console.log("Refactoring ...")
+
+// initialize
+const project = new Project({});
+
+project.addSourceFilesAtPaths("src/gen/*/*.ts");
+const files = project.getSourceFiles("src/gen/*/api.ts")
+
+files.map((file) => {
```

Comment:
Can we have a test for that?

Reply:
Just removed this functionality - no tests needed anymore